common/content/buffer.js

```diff
@@ -764,12 +764,6 @@ const Buffer = Module("buffer", {
             win.scrollBy(0, win.innerHeight / 2 * direction);
     },
 
-    _scrollByScrollSize: function _scrollByScrollSize(count, direction) {
-        if (count > 0)
-            options["scroll"] = count;
```

Comment:
without this function we're not setting `optoins['scroll']` anywhere. I'm ok with moving this if statement into `scrollByScrollSize`. However, because it was exported previously as `direction, count`, it seems like a breaking change.